.github/workflows/release.yml

```diff
@@ -66,7 +66,7 @@ jobs:
           PKG_SUFFIX=".tar.gz" ; case "${{ matrix.job.target }}" in *-pc-windows-*) PKG_SUFFIX=".zip" ;; esac;
           PKG_BASENAME="${PROJECT_NAME}-v${{ steps.project_version.outputs.PROJECT_VERSION }}-${{ matrix.job.target }}"
           PKG_PATH="${PKG_BASENAME}${PKG_SUFFIX}"
-          echo "PKG_PATH=${PKG_PATH}" >> "${GITHUB_ENV}"
+          echo "PKG_PATH=${PKG_PATH}" >> "${GITHUB_OUTPUT}"
```

Comment:
https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-output-parameter

https://github.com/ikanago/omekasy/blob/4f3c10d6a02320bd8fb3893e8f74bd514f4040ec/.github/workflows/release.yml#L88-L90